wallet/src/actors/worker/error.rs

```diff
@@ -26,6 +26,8 @@ pub enum Error {
     Send(#[cause] futures::sync::mpsc::SendError<std::string::String>),
     #[fail(display = "node error: {}", _0)]
     Node(#[cause] failure::Error),
+    #[fail(display = "timeout error")]
```

Comment:
Maybe something as...
```suggestion
    #[fail(display = "JsonRPC timeout error")]
```